utils/cosmos/iscn/sign.ts

```diff
@@ -188,3 +188,30 @@ export async function signISCN(
   const res = await signingPromise
   return res as DeliverTxResponse
 }
+
+export async function updateISCNRecord(
+  signer: OfflineSigner,
+  address: string,
+  iscnId: string,
+  payload: ISCNSignPayload,
+) {
+  const signingClient = await getSigningClient()
+  await signingClient.connectWithSigner(network.rpcURL, signer)
+  const signingPromise = signingClient.updateISCNRecord(
+    address,
+    iscnId,
+    payload,
+    {
+      fee: {
+        gas: UPDATE_ISCN_GAS_FEE.toString(),
+        amount: [{
+            denom: DEFAULT_GAS_PRICE[0].denom,
+            amount: DEFAULT_GAS_PRICE[0].amount.toString(),
```

Comment:
need to multiply by GAS_FEE?

Reply:
updated [8d4556c](https://github.com/likecoin/app-like-co/pull/429/commits/8d4556c6191ad440b8dacb877a6a67c2501db64d)

Reply:
We can just use `signISCN` or `signISCNTx` with `iscnId` instead of making a new functions